.github/workflows/e2e.yml

```diff
@@ -183,13 +191,6 @@ jobs:
             ${{ runner.os }}-next-cache-${{ github.sha }}
             ${{ runner.os }}-next-cache-
 
-      - name: Restore build (frontend)
-        uses: actions/cache@v1
-        id: next-build
-        with:
-          path: dist
-          key: ${{ runner.os }}-next-build-${{ github.sha }}
```

Comment:
We don't rely on the dist folder anymore, it was not playing well with server side code coverage.